functions/export-csv.js

```diff
@@ -51,6 +51,10 @@ async function exportCsv(req, res) {
   });
 
   res.type("text/csv");
+  res.setHeader(
+    "Content-Disposition",
+    "attachment; filename=" + layer.name.en + ".csv"
```

Comment:
What happens if there are slashes or other special characters in the filename? Some ideas here:
https://stackoverflow.com/questions/8485027/javascript-url-safe-filename-safe-string

Reply:
> What happens if there are slashes or other special characters in the filename? Some ideas here:
> https://stackoverflow.com/questions/8485027/javascript-url-safe-filename-safe-string

/ changes to _. Would we like to replace any special character with _?

Reply:
I think "-" would be a more modern approach. Wdyt?

Reply:
> I think "-" would be a more modern approach. Wdyt?

Sounds good, made the change, please take a look again :)